media-gfx/graphviz/graphviz-4.0.0.ebuild

```diff
@@ -0,0 +1,258 @@
+
+# There are several other tools in /tclpkg:
+# gdtclft, tcldot, tclhandle, tclpathplan, tclstubs ; enabled with: --with-tcl
+# tkspline, tkstubs ; enabled with: --with-tk
+
+# And the commands (/cmd):
+# - dot, dotty, gvedit, gvpr, lefty, lneato, smyrna, tools/* :)
+#   sci-libs/gts can be used for some of these
+# - lefty:
+#   needs Xaw and X to build
+# - gvedit (via 'qt5'):
+#   based on ./configure it needs qt-core and qt-gui only
+# - smyrna : experimental opengl front-end (via 'smyrna')
+#   currently disabled -- it segfaults a lot
+#   needs x11-libs/gtkglext, gnome-base/libglade, media-libs/freeglut
+#   sci-libs/gts, x11-libs/gtk.  Also needs 'gtk','glade','glut','gts' and 'png'
+#   with flags enabled at configure time
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+
+	java-pkg-opt-2_pkg_setup
+}
+
+src_prepare() {
+	default
+
+	# ToDo: Do the same thing for examples and/or
+	#       write a patch for a configuration-option
+	#       and send it to upstream
+	# note - the longer sed expression removes multi-line assignments that are extended via '\'
+	if ! use doc ; then
+		find . -iname Makefile.am \
+			| xargs sed -i -e '/^\(html\|pdf\)_DATA.*\\[[:space:]]*$/{:m;N;s/\\\n//;tm;d}' \
+				-e '/^\(html\|pdf\)_DATA/d' || die
+	fi
+
+	# Nuke the dead symlinks for the bindings
+	sed -i -e '/$(pkgluadir)/d' tclpkg/gv/Makefile.am || die
+
+	use java && append-cppflags $(java-pkg_get-jni-cflags)
+
+	eautoreconf
+}
+
+src_configure() {
+	local myconf=(
+		# Speeds up the libltdl configure
+		--cache-file="${S}"/config.cache
+		--enable-ltdl
+		$(use_with cairo pangocairo)
+		$(use_with examples demos)
+		$(use_with devil)
+		$(use_with gtk2 gdk)
+		$(use_with gtk2 gdk-pixbuf)
+		$(use_with gtk2)
+		$(use_with gts)
+		$(use_with qt5 qt)
+		$(use_with lasi)
+		$(use_with pdf poppler)
+		$(use_with postscript ghostscript)
+		$(use_with svg rsvg)
+		$(use_with webp)
+		$(use_with X x)
+		$(use_enable lefty)
+		--with-digcola
+		--with-fontconfig
+		--with-freetype2
+		--with-ipsepcola
+		--with-libgd
+		--with-sfdp
+		--without-ming
+		# new/experimental features, to be tested, disable for now
+		--without-glitz
+		--without-ipsepcola
+		--without-smyrna
+		--without-visio
+		# Bindings:
+		$(use_enable guile)
+		$(use_enable java)
+		$(use_enable perl)
+		$(use_enable python python3)
+		$(use_enable ruby)
+		$(use_enable tcl)
+		--disable-go
+		--disable-io
+		--disable-lua
+		--disable-ocaml
+		--disable-php
+		--disable-python
+		--disable-r
+		--disable-sharp
+		# libtool file collision, bug #276609
+		--without-included-ltdl
+		--disable-ltdl-install
+	)
+	econf "${myconf[@]}"
+}
+
+src_install() {
+	default
+
+	find "${ED}" -name '*.la' -delete || die
+
+	einstalldocs
+
+	use python && python_optimize \
+		"${D}"$(python_get_sitedir) \
+		"${ED}"/usr/$(get_libdir)/graphviz/python3
+}
+
+pkg_postinst() {
+	# This actually works if --enable-ltdl is passed
+	# to configure
+	dot -c || die
```

Comment:
nix the comment and replace it with
`# we need to register all plugins before they become usable`